tests/MyGame/Example/ArrayStruct.py

```diff
@@ -20,53 +20,59 @@ def Init(self, buf, pos):
     # ArrayStruct
     def A(self): return self._tab.Get(flatbuffers.number_types.Float32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(0))
     # ArrayStruct
-    def B(self): return [self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(4 + i * 4)) for i in range(15)]
+    def B(self, j = None):
+        if j==None:
```

Comment:
is that the pythonic way to check None? I thought it was `is `None` or something?

Reply:
yep it is, I have habits from c++ still, but I'll make a quick change

Reply:
changed with the latest commit